content/plugins/environment-plugin.md

```diff
@@ -2,23 +2,67 @@
 title: environment-plugin
 contributors:
   - simon04
+  - einarlove
 ---
 
-The environment-plugin is a shorthand for using the [define-plugin](/plugins/define-plugin)
-on [`process.env`](https://nodejs.org/api/process.html#process_process_env) keys.
+The EnvironmentPlugin is a shorthand for using the [DefinePlugin](/plugins/define-plugin) on [`process.env`](https://nodejs.org/api/process.html#process_process_env) keys.
+```js
+new webpack.DefinePlugin({
+  'process.env.NODE_ENV': JSON.stringify(process.env.NODE_ENV)
+})
+```
+can be replaced with
+```js
+new webpack.EnvironmentPlugin([ 'NODE_ENV' ])
+```
+
 
-**Example:**
+## How to use
+The EnvironmentPlugin accepts either an array of keys or an object with default values.
 
-```javascript
-new webpack.EnvironmentPlugin([
-  'NODE_ENV'
-])
+```js
+new webpack.EnvironmentPlugin(['NODE_ENV', 'VERSION'])
 ```
 
-This is equivalent to the following define-plugin application:
+When given an object, the values are used as defaults for when they are not defined in `process.env`.
 
-```javascript
-new webpack.DefinePlugin({
-  'process.env.NODE_ENV': JSON.stringify(process.env.NODE_ENV)
+```js
+new webpack.EnvironmentPlugin({
+  NODE_ENV: 'development',
+  VERSION: 33,
```

Comment:
You already have `import` etc. in a lot of places so going with the current standard doesn't feel too bad. The gotcha is that you will need to process through Babel or Buble to support older browsers, but that's a separate problem.